### ISSUE TITLE
cannot get to the admin page

### ISSUE THREAD

**Issue by GrandM1961** [daylightstudio/FUEL-CMS#602]
I have followed everything to setup Fuel and after everything done I want to goto localhost/fuel/fuel/ as it directs me and a 404 appears Not found. When I look into the index.php it says this: <?php header('Location: start'); exit();

No idea where to look for start as a location, so I got stuck even before I can begin.
Some pointers would be nice, thank you.

**Comment by conroywong** [comment]
Seems you haven't configured well php environment in your computer. Suggested you to use xampp or wamp for beginners.

**Comment by GrandM1961** [comment]
Well it does exactly the same on that as well, so out of the box I get the same error, so nothings wrong with my php environment.

**Comment by daylightstudio** [comment]
So to be clear, you've setup the installation in a subfolder called "fuel" correct? If so, did you add that folder to the .htaccess like so?
```
<IfModule mod_rewrite.c>
	RewriteEngine On
	RewriteBase /fuel/
```
If so, did you enable the admin in `fuel/application/config/MY_fuel.php` `$config['admin_enabled']`?

**Comment by GrandM1961** [comment]
Yes i did put it in was my mistake, but now it throws a bunch of errors. I give up
